src/dos/drive_iso.cpp

```diff
@@ -192,7 +192,7 @@ int isoDrive::UpdateMscdex(char drive_letter, const char *path, uint8_t &sub_uni
 		CDROM_Interface *cdrom = new CDROM_Interface_Image(sub_unit);
 		char pathCopy[CROSS_LEN];
 		safe_strcpy(pathCopy, path);
-		if (!cdrom->SetDevice(pathCopy, 0)) {
+		if (!cdrom->SetDevice(pathCopy)) {
 			CDROM_Interface_Image::images[sub_unit] = oldCdrom;
 			delete cdrom;
 			return 3;
```

Comment:
+1

Reply:
It's some ugly error codes that gets later used as an output parameter to a constructor that can fail (yuck).

Yeah I don't like it either but I don't want to go re-factoring the .iso implementation just because I changed a single line here.

Reply:
> Yeah I don't like it either but I don't want to go re-factoring the .iso implementation just because I changed a single line here.

Fair enough.